src/collective/sitter/browser/static/collective.sitter.css

```diff
@@ -1,3 +1,3 @@
-.honeypot {
+#formfield-form-widgets-homepage {
```

Comment:
I do not want to tell spammers that this is a honeypot. So I do not want to rename the whole field. How can I add an additional class to a z3c formfield?

Reply:
```
    def updateWidgets(self):
        super().updateWidgets()
        self.fields['homepage'].widget.addClass("honeypot")
```

Reply:
This is not working :-(

```
self.fields['homepage'].widget
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: 'Field' object has no attribute 'widget'
```
`self.widgets['homepage'].addClass("honeypot")` is not working either because it adds the class only to input field. So label will be visible.